src/control/server/mgmt_system.go

```diff
@@ -0,0 +1,123 @@
+//
+// (C) Copyright 2018-2019 Intel Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+// The Government's rights to use, modify, reproduce, release, perform, display,
+// or disclose this software are subject to the terms of the Apache License as
+// provided in Contract No. 8F-30005.
+// Any reproduction of computer software, computer software documentation, or
+// portions thereof marked with this legend must also reproduce the markings.
+//
+
+package server
+
+import (
+	"github.com/golang/protobuf/proto"
+	"github.com/pkg/errors"
+	"golang.org/x/net/context"
+
+	mgmtpb "github.com/daos-stack/daos/src/control/common/proto/mgmt"
+)
+
+// systemStop sends multicast KillRank gRPC requests to system membership list.
+func (svc *mgmtSvc) systemStop(ctx context.Context, leader *IOServerInstance) error {
+	// TODO: inhibit rebuild on pool services, parallelise and make async.
+	for _, member := range svc.members.GetMembers() {
+		svc.log.Debugf("MgmtSvc.systemStop murder member %+v (not implemented)\n",
```

Comment:
While I personally find the use of "murder" in this context darkly amusing, we probably shouldn't use it in professional software. Hate to be the no-fun police...

For the record, no issue with the use of "kill", though. There is lots of precedent for that term.

Reply:
this was copying from core code, where I also found it amusing. I had meant to take it out before finalising :-)